client/src/components/Navbar/Navbar.css

```diff
@@ -1,34 +1,33 @@
-/* Navbar.css */
-.navbar {
-  background-color: rgb(4, 16, 36); /* Set your desired background color */
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding: 10px 20px;
-}
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
 
-.logo a {
-  color: #fff; /* Set the text color for the logo */
-  text-decoration: none;
-  font-size: 24px; /* Adjust the font size as needed */
-  font-weight: bold;
+/* Custom keyframe animation for fadeIn */
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+  to {
```

Comment:
Try to add other` @keyframe` as well and here you can add other bunch of transitions also.